packager/VisibilityChecker.cc

```diff
@@ -423,6 +423,18 @@ class VisibilityCheckerPass final {
         return tree;
     }
 
+    ast::ExpressionPtr preTransformClassDef(core::Context ctx, ast::ExpressionPtr tree) {
+        auto &original = ast::cast_tree_nonnull<ast::ClassDef>(tree);
+        auto ait = original.ancestors.begin();
+        // if this is a class, then the first ancestor is the parent class
+        if (original.kind == ast::ClassDef::Kind::Class && !original.ancestors.empty()) {
+            *ait = ast::TreeMap::apply(ctx, *this, std::move(*ait));
+            ++ait;
+        }
```

Comment:
Not sure what we're using iterators for here 🤔 

maybe this started as a loop?

```suggestion
        if (original.kind == ast::ClassDef::Kind::Class && !original.ancestors.empty()) {
            auto &superClass = original.ancestors[0];
            superClass = ast::TreeMap::apply(ctx, *this, std::move(superClass));
        }
```

Reply:
ah yeah sorry I just pulled this from autogen, we only need the first one nvm